stylesheets/commons/Panels.less

```diff
@@ -83,11 +83,11 @@ html .panel-box-heading {
 			margin-right: 0.75rem;
 			transform-origin: center;
 			transition: transform 0.2s linear;
+		}
 
-			@media ( hover: hover ) {
-				&:hover {
-					opacity: 1;
-				}
+		@media ( hover: hover )  {
+			&:hover .panel-box-heading-icon {
```

Comment:
Not nice to have to not be able to use the parent selector for the -icon, but such is life (using it will create not the intended selector)